package_policies/R8.md

```diff
@@ -0,0 +1,7 @@
+**R8.** Each package should provide pre-processor macros for version
```

Comment:
```suggestion
Each package should provide pre-processor macros that allow for version
```

Reply:
Why just "allow"?  With careful usage of `git describe`, you can produce a useful and meaningful version description for version of your code.  (But that gets more complicated for multi-repo projects but you can still do it.)

Reply:
@bartlettroscoe Are you on the call?

The idea is to have some macros, but we leave the package to decide how to do that.

Also, this is at the level of C/C++/Fortran, we don't assume `git`.

Reply:
@bartlettroscoe - the primary purpose of this policy is to enable usage of type:

https://github.com/trilinos/Trilinos/blob/master/packages/amesos2/src/Amesos2_Superludist_FunctionMap.hpp#L137